stripe/src/main/java/com/stripe/android/model/Token.java

```diff
@@ -17,11 +17,12 @@
 public class Token implements StripePaymentSource {
```

Comment:
The documentation of the `Token` class should be updated. It's no longer just a card. I'd also link to the docs for Token, which is probably a good place to borrow text from. https://stripe.com/docs/api#tokens